### PR TITLE
HERO url dependent on HERO access

### PR DESCRIPTION
**Pull request by rabstejnek**

### CODE REVIEW

File: hawc/apps/myuser/views.py [hunk 1]
```diff
@@ -91,6 +91,10 @@ def get_context_data(self, **kwargs):
         )
         return context
 
+    def form_valid(self, form):
```
Comment (rabstejnek) [inline hawc/apps/myuser/views.py:94]: Session is not easily accessible from a post_save signal, so I update the session variable here instead.

**Comment by rabstejnek** [comment]
The only other place where the url is determined is [here](https://github.com/shapiromatron/hawc/blob/eed910c723bfd716b080386b5348caa347fbd659/frontend/lit/Reference.js#L9).
This is called within [DataPivotExtension](https://github.com/shapiromatron/hawc/blob/eed910c723bfd716b080386b5348caa347fbd659/frontend/summary/dataPivot/DataPivotExtension.js#L76) and [HeatmapDatastore](https://github.com/shapiromatron/hawc/blob/eed910c723bfd716b080386b5348caa347fbd659/frontend/summary/summary/heatmap/HeatmapDatastore.js#L394).
How would you suggest handling it? We can pass in the hero_access variable into these constructors but we'd be complicating these classes with an argument that has a very limited use case in the scope of these visuals.

**Comment by shapiromatron** [comment]
revisit using `window.localStorage` to set flag on user login and profile update. then we can dynamically update the hero url with private if they have access

**Comment by rabstejnek** [comment]
Closing in favor of #398, which uses the `window.localStorage` approach.

**Pull request closed without merging**